### DISCUSSION TITLE
Can I have a list without staring a repo

### DISCUSSION THREAD

**Discussion by dprosper** [community/community#8842] [Other Feature Feedback, Questions, & Ideas]
I like the idea of creating private/public list for repos, but I don't necessaringly want to star all repos added to my lists.  How can I do that?

**Comment by brianlovin** [comment]
Right now it's not possible to add a repo to a list without first starring it. Could you share more about the scenario you're in where you'd want those uncoupled? We've seen a few requests like this, so I just want to make sure we understand as much as possible before making any changes 😊

**Reply by dprosper** [reply]
I view starring a repo as something you do when you want to promote that repo to other users.  

I find very often that I uncover a repo that I would like to go back and explore in more details, but I don't know enough about it to promote it.  In some cases it may be a repo that I specifically don't want to promote (that would be a rare case).  Today I keep a list in a markdown file, but it would be more convenient to add it to a custom list.

**Reply by brianlovin** [reply]
Got it, thanks for the context!

**Reply by esdnm** [reply]
> Got it, thanks for the context!

hey, what's the current plan

**Reply by brianlovin** [reply]
This change isn't on our roadmap right now. Thanks for checking back in 😊

**Reply by marcocaldera** [reply]
In general I always prefer to group repositories to keep things organised. It's the main reason why I didn't fully switched to github yet. Lists sound a cool feature, I don't understand why I should star my repo if I want to put it on a list (and actually if I create a list directly while adding a repo I'm able to insert the repo in the new list without starring it. It sounds like the feature is there! Just to be unlocked!)

**Reply by Andrew6rant** [reply]
I wanted to create a list containing a number of my own repositories, but it feels weird to have to star them to do it.

**Reply by GiorgosXou** [reply]
**Why** should I star a project that I don't know yet [if it works](https://github.com/AceLewis/my_first_calculator.py),  [if that project goes against my morals](https://imgur.com/EDmqt8g), [contains sensitive informations *(that can be used againt my free will)*](https://github.com/cirosantilli/china-dictatorship) or [that I just don't want others to know about](https://github.com/Derfirm/pornhub-api), or whatever, **just for the sake of categorization?**

**Reply by GiorgosXou** [reply]
What if I just want *(like on YouTube)* to use a `watch later` list for projects I want to check later? Should I star those too?

**Reply by 212sheyik** [reply]
- [ ]

**Reply by 532910** [reply]
> scenario you're in where you'd want those uncoupled

Star is a like (❤️), it's a positive assessment, it means I like this repo. It has nothing common with a list, that is a collection. I would not like to star repo, but would like to remember it, and the current implementation of the lists is competently unusable for this, so I have to bookmark them in browser.

**Comment by cceyda** [comment]
did this get implemented like a couple hours ago? or have I ran into a bug?

**Reply by Andrew6rant** [reply]
I think it may be a bug. I am able to add a repo to a list without starring it only if that repo is one of my current pinned items, or if the list is created at the same time the repo is added (by using the "create list" button)

**Reply by cceyda** [reply]
hmm, it wasn't one of those cases for me. It happened when I created a new list and added at the same time.
posting the repo that it happened for here if github team wants to debug: https://github.com/stars/cceyda/lists/lists-of-things

**Reply by Shaikhrazagulam** [reply]
You are name

**Reply by GiorgosXou** [reply]
happened to me a few times too, but idk why. Bug for sure.

**Reply by will8683** [reply]
> did this get implemented like a couple hours ago? or have I ran into a bug?

Bug duh

**Comment by mjraha592** [comment]
> Right now it's not possible to add a repo to a list without first starring it. Could you share more about the scenario you're in where you'd want those uncoupled? We've seen a few requests like this, so I just want to make sure we understand as much as possible before making any changes 😊 

Please, all the new numbers have been transferred to Mjraha592 and the organizations will be updated. Thank you

**Comment by carlosmoliv** [comment]
> I wanted to create a list containing a number of my own repositories, but it feels weird to have to star them to do it.

Exactly, it'd be useful for me too.

**Comment by SalmanAhmrd** [comment]
Awsm

**Comment by SalmanAhmrd** [comment]
Yes

**Comment by will8683** [comment]
> I like the idea of creating private/public list for repos, but I don't necessaringly want to star all repos added to my lists. How can I do that?

Your smart

**Comment by Afek-Sakaju** [comment]
Agreeing with the original poster of the thread discussion, it would be helpful to have lists to organize repositories in a way that makes sense. The "Favorites" list is a separate entity and shouldn't be tied to the lists feature, in my opinion.

**Comment by SantiagoRR2004** [comment]
I also agree with the idea of separating lists from stars. This topic also appears in https://github.com/orgs/community/discussions/9067 and https://github.com/orgs/community/discussions/125708.

**Comment by csalmeida** [comment]
Appreciate this being considered as a feature.

Being able to create list of our own repositories (private and public) would be a nice feature, personally I experiment a lot and tend to version track it using `git`. These add up quickly and a lot of the times these repositories are related, it would be nice to be able to group them somehow so that they are easier to find if I don't remember the exact name.

Currently I feel like to find a repository I worked on a while ago I have to resort to a mix of search or scrolling until it shows up. 

Thanks for looking into this! 🙏
